Wikipedia/Legacy Data Migration/WMFLegacyImageDataMigration.swift

```diff
@@ -23,20 +23,8 @@ public class WMFLegacyImageDataMigration : NSObject {
     /// Image controller where data will be migrated.
     let imageController: WMFImageController
 
-    /// List of saved pages which is saved when the tasks are finished processing.
```

Comment:
Looks like a lot of refactoring in this class beyond the scope of the ticket. I didn't really see anything about the reasoning for the changes in the PR comments or the commit logs. 

I like removing code, but a little worried about these changes since they seem a little invasive. Do you have some rationale for these or is this something we can factor out to be reviewed separately?